tcms/rpc/tests/test_attachment.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init, invalid-name, objects-update-used
+
+from tcms.rpc.tests.utils import APITestCase
+from tcms.tests.factories import TestPlanFactory
+
+class TestRemoveAttachment(APITestCase):
```

Comment:
Missing test class for permissions

Reply:
Yeah, about that. I originally wanted to have the permissions test as well, but I've been running into some issues. I would need the permissions for `attachments.add_attachment` to add an attachment as well as `attachments.remove_attachment` to remove it, but I don't see a way to have both permissions simultaneously.

Reply:
Unless I'm approaching it from the wrong angle.

Reply:
Option 1) use utils.add_attachment() directly (see TestPlan.add_attachment method) - it calls another view which takes care of the actual operation and it only needs the user to be logged in.

Option 2) use user_should_have_perm() during setup to grant the additional permissions for adding and listing attachments.See how it is used in existing tests.